src/material-experimental/mdc-slider/slider.ts

```diff
@@ -211,19 +223,25 @@ export class MatSlider implements AfterViewInit, OnDestroy {
     return thumb === Thumb.END ? knobs[knobs.length - 1] : knobs[0];
   }
 
+    /** Gets the slider knob HTML element of the given thumb. */
+    _getValueIndicatorTextElement(thumb: Thumb): HTMLElement {
+      const elements = this._valueIndicatorTextElements.toArray().map(e => e.nativeElement);
```

Comment:
This seems a little inefficient since you don't have to map all the text elements only take one out. You could do something like this instead:

```
const element = thumb === Thumb.END ? this._valueIndicatorTextElements.last : this._valueIndicatorTextElements.first;
return element.nativeElement;
```